### ISSUE TITLE
Bootgrid: command-info requires get instead of info URL

### ISSUE THREAD

**Issue by 8191** [opnsense/core#5478]
In the OPNsense UIBootgrid, the "info" action is only wired when a "get" URL is set:

https://github.com/opnsense/core/blob/88334b838a13c7083f5ad29044ff65d4bb5f14ba/src/opnsense/www/js/opnsense_bootgrid_plugin.js#L83-L86

Is this intended; or should it require the "info" URL, which is later actually used in the event handler?

https://github.com/opnsense/core/blob/88334b838a13c7083f5ad29044ff65d4bb5f14ba/src/opnsense/www/js/opnsense_bootgrid_plugin.js#L351-L354

**Comment by 8191** [comment]
"command-info" is not used at all so far, as I can see. So should have no impact to change.

**Comment by AdSchellevis** [comment]
It's used in
https://github.com/opnsense/plugins/blob/16f3522d08d30919b17e66bdec38352ef4c75208/net/zerotier/src/opnsense/mvc/app/views/OPNsense/Zerotier/index.volt#L48
 https://github.com/opnsense/plugins/blob/16f3522d08d30919b17e66bdec38352ef4c75208/net/zerotier/src/opnsense/mvc/app/views/OPNsense/Zerotier/index.volt#L104 


Maybe also in some private repo's but I'm not 100% sure about that.

**Comment by 8191** [comment]
> It's used in
> https://github.com/opnsense/plugins/blob/16f3522d08d30919b17e66bdec38352ef4c75208/net/zerotier/src/opnsense/mvc/app/views/OPNsense/Zerotier/index.volt#L48
> https://github.com/opnsense/plugins/blob/16f3522d08d30919b17e66bdec38352ef4c75208/net/zerotier/src/opnsense/mvc/app/views/OPNsense/Zerotier/index.volt#L104

You're totally right. I grep'ed for `command-info` which is not really useful in that case. ;-)

What's your opinion in changing the `requires: ['get']` to `requires: ['info']` in
https://github.com/opnsense/core/blob/88334b838a13c7083f5ad29044ff65d4bb5f14ba/src/opnsense/www/js/opnsense_bootgrid_plugin.js#L83-L86

**Comment by AdSchellevis** [comment]
> What's your opinion in changing the requires: ['get'] to requires: ['info'] in

Sounds like a good plan indeed.

**Comment by agh1467** [comment]
@AdSchellevis I forgot to use the 'closes' tag in the commit. Can you close this issue manually instead?

**Comment by AdSchellevis** [comment]
@agh1467 sure :)
